src/routes/(console)/project-[project]/functions/function-[function]/deploymentDomains.svelte

```diff
@@ -8,18 +8,21 @@
 </script>
 
 <div class="u-flex u-gap-4 u-cross-center">
-    <a
-        href={`http://${domain.rules[0].domain}`}
-        target="_blank"
-        class="u-flex u-gap-4 u-cross-center">
-        <Trim alternativeTrim>
-            <span class="link">
-                {domain.rules[0].domain}
-            </span>
-        </Trim>
-        <span class="icon-external-link" aria-hidden="true" />
-    </a>
-    {#if domain.rules.length > 1}
+    {#if domain && domain.rules && domain.rules.length > 0}
```

Comment:
can we make this a bit cleaner by having a variable in the script? 👀 

something like 
```js
$: length: domain?.rules?.length
```

then we can have more simple conditions 👍🏻